en/frames_rover/ackermann_rover.md

```diff
@@ -88,7 +62,7 @@ To get an overview of all parameters that are related to the Ackermann rover mod
 
 ### General Parameters
 
-These parameters affect the general behaviour of the rover. This will influence both auto and manual modes.
+These parameters affect the general behavior of the rover. This will influence both auto and manual modes.
```

Comment:
We use UK spelling "generally".
```suggestion
These parameters affect the general behaviour of the rover. This will influence both auto and manual modes.
```